expanding on TrustStore and KeyStore #4578 #4060 (#7015)
opensearch-project/documentation-website: _security/configuration/opensearch-keystore.md

`opensearch-keystore` is a utility script used to manage an OpenSearch keystore. An OpenSearch keystore provides a secure method of storing sensitive information, such as passwords and keys, used in an OpenSearch cluster. The script allows you to securely create, list, add, and remove settings. It is included in the OpenSearch distribution. 

> {: .note} This is separate from the keystore and truststore used to store TLS certificates in JKS or PKCS12/PFX format for the transport and HTTP layer. The other keystore is outlined in the [Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#keystore-and-truststore-files) documentation.

Reviewer: ```suggestion
This is separate from the keystore and truststore used to store TLS certificates in JKS or PKCS12/PFX format for the transport and HTTP layer. The other keystore is outlined in the [keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#keystore-and-truststore-files) documentation.
{: .note} 
```

> {: .note} This is separate from the keystore and truststore used to store TLS certificates in JKS or PKCS12/PFX format for the transport and HTTP layer. The other keystore is outlined in the [Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#keystore-and-truststore-files) documentation.

Reviewer: ```suggestion
This is separate from the keystore and truststore used to store TLS certificates in JKS or PKCS12/PFX format for the transport and HTTP layer. The other keystore is outlined in the [Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#keystore-and-truststore-files) documentation.
{: .note} 
```

`opensearch-keystore` is a utility script used to manage an OpenSearch keystore. An OpenSearch keystore provides a secure method of storing sensitive information, such as passwords and keys, used in an OpenSearch cluster. The script allows you to securely create, list, add, and remove settings. It is included in the OpenSearch distribution. 

> This keystore is separate from the keystore and truststore used to store TLS certificates in JKS or PKCS12/PFX format for securing the transport and HTTP layers. For information about those keystores, refer to [Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#keystore-and-truststore-files).

Reviewer 2: ```suggestion
This keystore is separate from the keystore and truststore used to store TLS certificates in JKS or PKCS12/PFX format in order to secure the transport and HTTP layers. For information about those keystores, refer to [Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#keystore-and-truststore-files).
```

No response exists for this command. To confirm that the setting was deleted, use `opensearch-keystore list`.

> A complete list of secure settings which can be configured using `opensearch-keystore` is available at [(Advanced) Using encrypted password settings for SSL]({{site.url}}{{site.baseurl}}/security/configuration/tls/#advanced-using-encrypted-password-settings-for-ssl)

Reviewer: ```suggestion
A complete list of secure settings that can be configured using `opensearch-keystore` is available at [(Advanced) Using encrypted password settings for SSL]({{site.url}}{{site.baseurl}}/security/configuration/tls/#advanced-using-encrypted-password-settings-for-ssl).
```
